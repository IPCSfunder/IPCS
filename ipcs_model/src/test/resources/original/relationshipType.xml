<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<RELATIONSHIP_TYPE relationship_type_objid="1" name="FATHER"></RELATIONSHIP_TYPE>
</dataset>