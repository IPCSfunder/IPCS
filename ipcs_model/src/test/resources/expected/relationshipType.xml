<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<RELATIONSHIP_TYPE NAME="FATHER"></RELATIONSHIP_TYPE>
	<RELATIONSHIP_TYPE NAME="MOTHER"></RELATIONSHIP_TYPE>
</dataset>